client/dashboard/components/action-list/index.stories.tsx

```diff
@@ -0,0 +1,150 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Button, Icon } from '@wordpress/components';
+import { cog } from '@wordpress/icons';
+import ActionList from './index';
+
+const meta = {
+	title: 'client/dashboard/ActionList',
+	component: ActionList,
+	tags: [ 'autodocs' ],
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+	},
+} satisfies Meta< typeof ActionList >;
+
+export default meta;
+type Story = StoryObj< typeof meta >;
+
+export const Default: Story = {
+	args: {
+		children: (
+			<ActionList.ActionItem
```

Comment:
Should this be just ActionList.Item?

Reply:
I considered that earlier but I think it would be better to align the name with the Figma file. I'm fine with either 🙂